docs/hoon/library/1a.md

```diff
@@ -25,23 +25,37 @@ An atom.
 Source
 ------
 
-    ++  add                                                 ::  add
+    ++  add                                                
       ~/  %add
-      |=  [a/@ b/@]
+      |=  [a=@ b=@]
       ^-  @
       ?:  =(0 a)  b
       $(a (dec a), b +(b))
-
+::
 Examples
 --------
 
     > (add 2 2)
     4
+
     > (add 1 1.000.000)
     1.000.001
+
     > (add 1.333 (mul 2 2))
     1.337
 
+    > `@`'a'                                     ::`'a'` is an atom represented as a different data type.
+    97                                             We cast it to the aura `@` to see what its integer representation is.
+    > (add 'a' 4)                                
+    101
+    (add 'a' 'b')
+    195  
+
+    > (add 0xc 0xf)                              ::`0xc` and `0xf` are of the hexadecimal soft atom type (`@ux`).
```

Comment:
"soft atom type" is too imprecise a term for stdlib docs.  We should assume a basic familiarity with Hoon here; no explaining auras in the `add` gate reference.

Reply:
Also, short comments only for in-line comments; something like 4-5 words is probably already pushing it